### PR TITLE
RELATED: RAIL-779 Added unauthorized to known errors

### PR DESCRIPTION
**Pull request by vojty**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/89/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1339/ : SUCCESS in 1m 46s

**Comment by vojty** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/548/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/90/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/474/ : SUCCESS in 14m 12s

File: src/components/core/base/BaseVisualization.tsx [hunk 1]
```diff
@@ -6,39 +6,30 @@ import {
...
-                    message: intl.formatMessage({ id: 'visualization.ErrorMessageNoData' }),
-                    description: intl.formatMessage({ id: 'visualization.ErrorDescriptionNoData' })
-                },
-                [ErrorStates.UNKNOWN_ERROR]: {
-                    message: intl.formatMessage({ id: 'visualization.ErrorMessageGeneric' }),
-                    description: intl.formatMessage({ id: 'visualization.ErrorDescriptionGeneric' })
-                }
-            };
-            const errorProps = errorMap[errorMap.hasOwnProperty(error) ? error : ErrorStates.UNKNOWN_ERROR];
+            const errorProps = this.errorMap[this.errorMap.hasOwnProperty(error) ? error : ErrorStates.UNKNOWN_ERROR];
```
Comment (tomasreichmann) [inline src/components/core/base/BaseVisualization.tsx]: Some linter in one of our project complains about calling Object.prototype functions directly on hashes. IMHO, this is a bit stupid. You would have to call Object.prototype.hasOwnProperty.call(this.errorMap, error);
The reason is, you can have a hash like this:
{
  hasOwnProperty: true
}
Is this a valid concern or should we disable this lint rule next time we stumble upon it?

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/11/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1342/ : SUCCESS in 1m 27s
- https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/551/ : SUCCESS in 12m 21s

**Pull request merged**